docs/install/install-server/install-server.inc

```diff
@@ -61,104 +33,110 @@ Install the software
             sudo /opt/inmanta/bin/inmanta --help
 
             # Install PostgreSQL
             sudo apt-get install postgresql postgresql-client
 
 
-        Download the configuration file named ``inmanta.cfg`` (this name is arbitrary) in your virtual env:
+        Download the configuration files named ``inmanta.cfg`` and ``extensions.cfg`` (these names are arbitrary) in your virtual env:
 
         .. code-block:: sh
 
+            sudo mkdir /opt/inmanta/inmanta.d
             sudo apt-get install wget
             sudo wget -O /opt/inmanta/inmanta.cfg "https://raw.githubusercontent.com/inmanta/inmanta-core/master/misc/inmanta.cfg"
+            sudo wget -O /opt/inmanta/inmanta.d/extensions.cfg "https://raw.githubusercontent.com/inmanta/inmanta-core/master/misc/extensions.cfg"
 
 
-        If you want to use the dashboard you need to install it as well:
+        If you want to use the web-console and/or the dashboard you need to install it as well:
 
-        Get the pre-built package from `our github page <https://github.com/inmanta/inmanta-dashboard/packages/>`_ click on the latest release and then on the right hand side, under ``Assets``, you will see the compressed package. Download and extract it to your desired directory (preferably, on the same virtual env which was created earlier, in this case, /opt/inmanta). Next, open the ``inmanta.cfg`` file and at the bottom of the file, under ``[dashboard]`` section, change the ``path`` value to the ``dist`` directory of where you extracted the pre-built package. For instance:
+        Get the pre-built package from our `web-console <https://github.com/inmanta/web-console/packages/>`_ and/or `dashboard <https://github.com/inmanta/inmanta-dashboard/packages/>`_ github page. Click on the the package name to go to the package's main page, then on the right hand side under ``Assets``, you will see the compressed package. Download and extract it to your desired directory (preferably, on the same virtual env which was created earlier, in this case, /opt/inmanta). Next, open the ``inmanta.cfg`` file and at the bottom of the file, under the ``[dashboard]`` and/or ``[web-console]`` section, change the ``path`` value to the ``dist`` directory of where you extracted the pre-built package. For instance:
 
         .. code-block:: ini
 
-            path=/opt/inmanta/package/dist
+            path=/opt/inmanta/web-console/package/dist
 
 
-        Then the dashboard can be started using below command (please note, below command has to be run after completing the  :ref:`configure-server`) part:
+        Then the web-console/dashboard can be started using below command (please note, below command has to be run after completing the  :ref:`configure-server`) part:
 
         .. code-block:: bash
 
-            inmanta -vv -c /opt/inmanta/inmanta.cfg server
+            /opt/inmanta/bin/inmanta -vv -c /opt/inmanta/inmanta.cfg --config-dir /opt/inmanta/inmanta.d server
```

Comment:
All config files and directories are creates using `sudo`. So we will have to start the server as root as well.
```suggestion
            sudo /opt/inmanta/bin/inmanta -vv -c /opt/inmanta/inmanta.cfg --config-dir /opt/inmanta/inmanta.d server
```